encrypted-media-respec.html

```diff
@@ -4038,14 +4039,14 @@
             Potentially, this could reveal information about a user to another user of the same device, including potentially the <a def-id="origin">origins</a> that have used a particular Key System (i.e. sites visited) or even the content that has been decrypted using a Key System.
           </p>
           <p>If information stored by one origin affects the operation of the Key System for another origin, then potentially the sites visited or content viewed by a user on one site may be revealed to another, potentially malicious, site.</p>
-          <p>If information stored for one <a def-id="user-agent-profile"></a> on the client device affects the operation of the Key System for other <a def-id="user-agent-profile">user agent profiles</a>, or browsers, then potentially the sites visited or content viewed in one may be revealed by or correlatable with another <a def-id="user-agent-profile">user agent profiles</a> or browser.</p>
+          <p>If information stored for one <a def-id="browsing-profile"></a> on the client device affects the operation of the Key System for other <a def-id="browsing-profile">browsing profiles</a>, or browsers, then potentially the sites visited or content viewed in one may be revealed by or correlatable with another <a def-id="browsing-profile">browsing profiles</a> or browser. Note that different operating system user accounts are defined as different <a def-id="browsing-profile">browsing profiles</a>.</p>
```

Comment:
While accurate, the "Note..." sentence seems longer and less precise than necessary. Does adding ", including across operating system user accounts" to the previous sentence work?